Jenkinsfile

```diff
@@ -67,15 +67,30 @@ pipeline {
             }
         }
  
-        stage('Build Avalon') {
+        stage('Build Avalon Direct Model Components') {
             steps {
-                sh 'docker-compose -f ci/docker-compose-build.yaml build'
+                sh 'docker-compose -f ci/docker-compose-direct-model-build.yaml build'
+            }
+        }
+ 
+        stage('Run Avalon Direct Model Tests') {
+            steps {
+                sh 'INSTALL_TYPE="" ./bin/run_tests -d'
             }
         }
 
-        stage('Run Avalon Tests') {
+        stage('Build Avalon Proxy Model Components') {
             steps {
-                sh 'INSTALL_TYPE="" ./bin/run_tests'
+                sh 'docker-compose -f ci/docker-compose-proxy-model-build.yaml build'
+            }
+        }
+
+        stage('Run Avalon proxy Model Tests') {
+            steps {
+                sh '''
+                   ./scripts/start_fabric.sh -u
+                   INSTALL_TYPE="" ./bin/run_tests -p
+                   '''
```

Comment:
Do we need to terminate all the containers for peers, orderers at the end of job?